edxpipelines/pipelines/config/frontend_apps.yml

```
@@ -9,6 +9,7 @@ apps:
   - name: mitxpro-journals-frontend
     source_url: git@github.com:edx/journals-frontend.git
     branch: master
+    purge_cache: no
```

Can we change this for consistency with other Boolean values in our YAML?
```suggestion
    purge_cache: false
```

---

@jdmulloy change boolean value to make it consistent.